### DISCUSSION TITLE
Order boxes inside section

### DISCUSSION THREAD

**Discussion by PubliAlex** [OFFLINE-GmbH/oc-boxes-support#183] [Ideas and feature requests]
Hello,

I red on the doc that you can order sections the way you want by modifying boxes.yaml file in the theme directory, but I didn't see how you can order boxes inside a section.

I have sections with lots of boxes inside and some boxes are quite related so I would like to display them grouped, but it would make not sense to create another group for that.

Thank you

**Comment by tobias-kuendig** [comment]
Currently, the sections can be sorted, but the boxes inside that section are always sorted alphabetically. There are no plans to introduce an additional layer of grouping, so sections are all you've got to work with at the moment.

**Comment by PubliAlex** [comment]
Boxes inside section don't seems to be displayed alphabetically cause I have several boxes, having practically the same name that are far apart in the list

**Reply by tobias-kuendig** [reply]
This is fixed in v3.6.49, Partials are now sorted properly by name/handle.

There is also a new `order` property in the YAML to override the default order logic.

https://docs.boxes.offline.ch/concepts/box-configs.html#order
